orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -1134,8 +1134,14 @@ def run_step(model, input):
         _test_helpers.assert_values_are_close(ort_prediction, pt_prediction)
         _test_helpers.assert_values_are_close(ort_input.grad, pt_input.grad)
 
-@pytest.mark.parametrize("equation", ["s,se->se", "se,sc->sec", "se,se->s", "sec,sm->ecm",
-                                      "sec,ecm->sm", "ks,ksm->sm", "kes,ems->mek", "kes,ksm->ms"])
+# In PyTorch 1.11.0, there is issue during reduce node shape handling for exporter, so any sub-graph that
+# contains ReduceProd will fail to run, for example, "sec,sm->ecm", "sec,ecm->sm".
+# Currently move these cases out from the tests, will enable these tests again once the issue in PyTorch is fixed.
+einsum_equations = ["s,se->se", "se,sc->sec", "se,se->s", "ks,ksm->sm", "kes,ems->mek", "kes,ksm->ms"]
+if LooseVersion(torch.__version__) < LooseVersion('1.11.0'):
+    einsum_equations.append("sec,sm->ecm")
+    einsum_equations.append("sec,ecm->sm")
+@pytest.mark.parametrize("equation", einsum_equations)
```

Comment:
I think this is fine, but you could consider using `pytest.param("sec,sm->ecm", marks=pytest.mark.skipif(LooseVersion(torch.__version__) >= LooseVersion('1.11.0'), reason="torch 1.11.0 incompatible")),`.
Let's add a TODO here too.